README.md

```diff
@@ -1161,6 +1161,19 @@ Note that options in configuration file are just the same options aka switches u
 
 You can use `--ignore-config` if you want to disable all configuration files for a particular yt-dlp run. If `--ignore-config` is found inside any configuration file, no further configuration will be loaded. For example, having the option in the portable configuration file prevents loading of home, user, and system configurations. Additionally, (for backward compatibility) if `--ignore-config` is found inside the system configuration file, the user configuration is not loaded.
 
+### Specifying encoding of config files
+
+By default, config files are read by UTF-8.
+If you saved your config file in a different encoding than that, you may write any of these to the first line of the file:
+
+1. `# -*- coding: ENCODING -*-` (e.g. `# -*- coding: gbk -*-`)
+1. `# coding: ENCODING` (e.g. `# coding: shift-jis`)
+1. `# vi: set fileencoding=ENCODING` (e.g. `# vi: set fileencoding=euc-kr`)
```

Comment:
No need to over-complicate. Let's only support one of these. My preference is `# coding: ENCODING`

Reply:
Okay going with 2. (which is used in youtube-dl and was used in yt-dlp before syntax upgrade)